src/components/atoms/ImageInput/ImageInput.scss

```diff
@@ -26,6 +26,10 @@
       display: inline-block;
       padding: $image-input-padding--small;
     }
+
+    &--hidden {
+      display: none;
+    }
```

Comment:
Not necessary, there already is `mod--hidden` in `modifiers.scss` included globally

Reply:
fixed 65d922a